outputs.tf

```
@@ -3,3 +3,8 @@ output "efs_file_system_id" {
   description = "EFS File System ID"
   value       = try(aws_efs_file_system.this[0].id, "")
 }
+
+output "efs_backup_iam_role" {
+  description = "ARN of the IAM Role created to run AWS Backup"
+  value       = try(aws_iam_role.backup[0].id, "")
```

Would a null value not be better than an empty string if there's no IAM role?

```suggestion
  value       = try(aws_iam_role.backup[0].id, null)
```

---

the terraform-docs file was created and committed by the linter bot, I'm not.. super interested in troubleshooting our nannies at the moment, but I do agree on the null

---

ugh.

We've been disabling the linter bot slowly. I'll open an issue to disable it in this repo